<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurnIn_GUI</class>
 <widget class="QMainWindow" name="BurnIn_GUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Burn-In Controller v0.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="BurnIn_tab">
       <attribute name="title">
        <string>BurnIn</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>72</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Coming soon...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Test_tab">
       <attribute name="title">
        <string>Test Operation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Julabo</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Julabo cmd:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="JulaboTestCmd_line"/>
             </item>
             <item>
              <widget class="QPushButton" name="JulaboTestCmd_btn">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>FNAL Box</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Door magnet</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="MagON_btn">
               <property name="text">
                <string>Magnet ON</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="MagOFF_btn">
               <property name="text">
                <string>Magnet OFF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Dry air Flow</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="FLOWLow_btn">
               <property name="text">
                <string>LOW flow</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="FLOWHigh_btn">
               <property name="text">
                <string>HIGH flow</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>FNAL cmd:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="FNALTestCmd_line"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>CAEN PS</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>CAEN cmd:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="CAENTestCmd_line"/>
             </item>
             <item>
              <widget class="QPushButton" name="CAENTestCmd_btn">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>26</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExpert"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExpert">
   <property name="icon">
    <iconset>
     <normaloff>expert.png</normaloff>expert.png</iconset>
   </property>
   <property name="text">
    <string>Expert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>exit.jpeg</normaloff>exit.jpeg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>toggled(bool)</signal>
   <receiver>BurnIn_GUI</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
